### PR TITLE
feat: add monitoring profile for the validator

### PR DESCRIPTION
**Pull request by wo-o**
Add Prometheus, Grafana for monitoring kroma-validator

### CODE REVIEW

**Review by seolaoh — APPROVED** [review]

File: config/.DS_Store
Comment (ccrayz) [inline config/.DS_Store:1]: I'm wondering if there's a reason for that file to exist

Comment (0xHansLee) [reply config/.DS_Store:1]: Please remove this file and add `.DS_Store` in `.gitignore`

File: docker-compose.yml [hunk 1]
```diff
@@ -90,6 +90,37 @@ services:
     profiles:
       - prover
 
+  prometheus:
```
Comment (0xHansLee) [inline docker-compose.yml:93]: If this is for monitoring validator, do we need to add `depend_on` validator and node?

**Review by 0xHansLee — APPROVED** [review]
LGTM
It would be better to make branch for each PR for the next time :+1:

**Pull request merged**